gcslock.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
```

Comment:
If this is only sourced, not executed, I don't think you need #!/bin/sh, do you.